.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,10 @@
+- id: jupytext
```

Comment:
We need a copy of this hook using the system language so that it runs off the system version of JuPytext, happy to write it @JohnPaton . We can even reuse the node by taking advantage of some nice advanced YAML features.

Reply:
Why do we need that? Users can just do (incomplete example)
```yaml 
- repo: local
  hooks:
    language: system
    entry: jupytext
```
if they want that